README.md

```diff
@@ -104,12 +104,16 @@ There is no need to save manually.
 Address book data are saved in a file called `addressbook.txt` in the project root folder.
 You can change the location by specifying the file path as a program argument.
 
-Example: 
+Example:
 * `java seedu.addressbook.AddressBook mydata.txt`
```

Comment:
Add a newline above this line otherwise the Markdown won't render correctly on GitHub pages.